### ISSUE TITLE
Bug: Data from middleware is not available in dialogs (version 2.0.0b1)

### ISSUE THREAD

**Issue by t3m8ch** [Tishka17/aiogram_dialog#129]
main.py
```Python
import logging
import os

from aiogram import Router, F, Bot, Dispatcher
from aiogram.dispatcher.fsm.storage.memory import MemoryStorage
from aiogram_dialog import DialogManager, DialogRegistry

from dialog import dialog_1, Dialog1SG, Dialog2SG, dialog_2
from service import HelloWorldService

TOKEN = os.getenv("TOKEN")


async def di_middleware(handler, event, data):
    service = HelloWorldService()
    data["hello_world_service"] = service
    return await handler(event, data)


router = Router()


@router.message(F.text == "/dialog1")
async def on_start_command(_, dialog_manager: DialogManager, hello_world_service):
    print(hello_world_service)  # OK!
    await dialog_manager.start(state=Dialog1SG.state)


@router.message(F.text == "/dialog2")
async def on_start_command(_, dialog_manager: DialogManager, hello_world_service):
    print(hello_world_service)  # OK!
    await dialog_manager.start(state=Dialog2SG.state)


def main():
    logging.basicConfig(level=logging.INFO)

    bot = Bot(token=TOKEN)
    fsm_storage = MemoryStorage()
    dp = Dispatcher(storage=fsm_storage)

    dp.message.middleware(di_middleware)
    dp.callback_query.middleware(di_middleware)

    dp.include_router(router)
    registry = DialogRegistry(dp)
    registry.register(dialog_1)
    registry.register(dialog_2)

    dp.run_polling(bot)


if __name__ == "__main__":
    main()
```

dialog.py
```Python
from aiogram.dispatcher.fsm.state import StatesGroup, State
from aiogram_dialog import Dialog, Window, DialogManager
from aiogram_dialog.widgets.kbd import Button
from aiogram_dialog.widgets.text import Format, Const

from service import HelloWorldService


class Dialog1SG(StatesGroup):
    state = State()


class Dialog2SG(StatesGroup):
    state = State()


async def getter(dialog_manager: DialogManager, **kwargs):
    service: HelloWorldService = dialog_manager.data["hello_world_service"]  # NOT OK
    return {
        "hello_world_text": service.get_text()
    }


async def on_click(_, __, manager: DialogManager):
    service: HelloWorldService = manager.data["hello_world_service"]  # NOT OK
    print(service.get_text())


dialog_1 = Dialog(
    Window(
        Format("{hello_world_text}"),
        state=Dialog1SG.state,
        getter=getter,
    )
)

dialog_2 = Dialog(
    Window(
        Const("Click me!"),
        Button(Const("Click!"), "btn", on_click),
        state=Dialog2SG.state
    )
)
```

service.py
```Python
class HelloWorldService:
    def get_text(self) -> str:
        return "Hello world!"
```

When accessing the dialog_manager.data dictionary, a KeyError exception is raised:
```
KeyError: 'hello_world_service'
```

**Comment by t3m8ch** [comment]
requirements.txt
```
aiofiles==0.8.0
aiogram==3.0.0b2
aiogram-dialog==2.0.0b1
aiohttp==3.8.1
aiosignal==1.2.0
async-timeout==4.0.2
attrs==21.4.0
cachetools==4.2.4
charset-normalizer==2.0.12
frozenlist==1.3.0
idna==3.3
Jinja2==3.0.3
magic-filter==1.0.5
MarkupSafe==2.1.0
multidict==6.0.2
pydantic==1.9.0
Pygments==2.11.2
typing-extensions==4.1.1
yarl==1.7.2
```

Python version: 3.10

**Comment by t3m8ch** [comment]
Traceback when /dialog1:
```
<service.HelloWorldService object at 0x7f21bf971c30>
INFO:aiogram.event:Update id=840847703 is not handled. Duration 0 ms by bot id=1589390197
ERROR:aiogram.event:Cause exception while process update id=840847703 by bot id=1589390197
KeyError: 'hello_world_service'
Traceback (most recent call last):
  File "/mnt/Trash/artem/PythonDev/aiogram_dialog_middlwares_issue/venv/lib64/python3.10/site-packages/aiogram/dispatcher/dispatcher.py", line 238, in _process_update
    response = await self.feed_update(bot, update, **kwargs)
  File "/mnt/Trash/artem/PythonDev/aiogram_dialog_middlwares_issue/venv/lib64/python3.10/site-packages/aiogram/dispatcher/dispatcher.py", line 104, in feed_update
    response = await self.update.wrap_outer_middleware(self.update.trigger, update, kwargs)
  File "/mnt/Trash/artem/PythonDev/aiogram_dialog_middlwares_issue/venv/lib64/python3.10/site-packages/aiogram/dispatcher/middlewares/error.py", line 24, in __call__
    return await handler(event, data)
  File "/mnt/Trash/artem/PythonDev/aiogram_dialog_middlwares_issue/venv/lib64/python3.10/site-packages/aiogram/dispatcher/middlewares/user_context.py", line 23, in __call__
    return await handler(event, data)
  File "/mnt/Trash/artem/PythonDev/aiogram_dialog_middlwares_issue/venv/lib64/python3.10/site-packages/aiogram/dispatcher/fsm/middleware.py", line 39, in __call__
    return await handler(event, data)
  File "/mnt/Trash/artem/PythonDev/aiogram_dialog_middlwares_issue/venv/lib64/python3.10/site-packages/aiogram_dialog/manager/manager_middleware.py", line 35, in __call__
    return await handler(event, data)
  File "/mnt/Trash/artem/PythonDev/aiogram_dialog_middlwares_issue/venv/lib64/python3.10/site-packages/aiogram_dialog/context/intent_filter.py", line 67, in __call__
    return await handler(event, data)
  File "/mnt/Trash/artem/PythonDev/aiogram_dialog_middlwares_issue/venv/lib64/python3.10/site-packages/aiogram/dispatcher/event/telegram.py", line 239, in trigger
    return await wrapped_inner(event, kwargs)
  File "/mnt/Trash/artem/PythonDev/aiogram_dialog_middlwares_issue/venv/lib64/python3.10/site-packages/aiogram/dispatcher/event/handler.py", line 43, in call
    return await wrapped()
  File "/mnt/Trash/artem/PythonDev/aiogram_dialog_middlwares_issue/venv/lib64/python3.10/site-packages/aiogram/dispatcher/dispatcher.py", line 205, in _listen_update
    return await self.propagate_event(update_type=update_type, event=event, **kwargs)
  File "/mnt/Trash/artem/PythonDev/aiogram_dialog_middlwares_issue/venv/lib64/python3.10/site-packages/aiogram/dispatcher/router.py", line 128, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
  File "/mnt/Trash/artem/PythonDev/aiogram_dialog_middlwares_issue/venv/lib64/python3.10/site-packages/aiogram/dispatcher/router.py", line 124, in _wrapped
    return await self._propagate_event(
  File "/mnt/Trash/artem/PythonDev/aiogram_dialog_middlwares_issue/venv/lib64/python3.10/site-packages/aiogram/dispatcher/router.py", line 144, in _propagate_event
    response = await router.propagate_event(update_type=update_type, event=event, **kwargs)
  File "/mnt/Trash/artem/PythonDev/aiogram_dialog_middlwares_issue/venv/lib64/python3.10/site-packages/aiogram/dispatcher/router.py", line 128, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
  File "/mnt/Trash/artem/PythonDev/aiogram_dialog_middlwares_issue/venv/lib64/python3.10/site-packages/aiogram/dispatcher/router.py", line 124, in _wrapped
    return await self._propagate_event(
  File "/mnt/Trash/artem/PythonDev/aiogram_dialog_middlwares_issue/venv/lib64/python3.10/site-packages/aiogram/dispatcher/router.py", line 137, in _propagate_event
    response = await observer.trigger(event, **kwargs)
  File "/mnt/Trash/artem/PythonDev/aiogram_dialog_middlwares_issue/venv/lib64/python3.10/site-packages/aiogram/dispatcher/event/telegram.py", line 239, in trigger
    return await wrapped_inner(event, kwargs)
  File "/mnt/Trash/artem/PythonDev/aiogram_dialog_middlwares_issue/main.py", line 17, in di_middleware
    return await handler(event, data)
  File "/mnt/Trash/artem/PythonDev/aiogram_dialog_middlwares_issue/venv/lib64/python3.10/site-packages/aiogram/dispatcher/event/handler.py", line 43, in call
    return await wrapped()
  File "/mnt/Trash/artem/PythonDev/aiogram_dialog_middlwares_issue/main.py", line 26, in on_start_command
    await dialog_manager.start(state=Dialog1SG.state)
  File "/mnt/Trash/artem/PythonDev/aiogram_dialog_middlwares_issue/venv/lib64/python3.10/site-packages/aiogram_dialog/manager/manager.py", line 114, in start
    await self._start_normal(state, data)
  File "/mnt/Trash/artem/PythonDev/aiogram_dialog_middlwares_issue/venv/lib64/python3.10/site-packages/aiogram_dialog/manager/manager.py", line 158, in _start_normal
    await self._dialog().show(self)
  File "/mnt/Trash/artem/PythonDev/aiogram_dialog_middlwares_issue/venv/lib64/python3.10/site-packages/aiogram_dialog/dialog.py", line 115, in show
    new_message = await window.render(self, manager)
  File "/mnt/Trash/artem/PythonDev/aiogram_dialog_middlwares_issue/venv/lib64/python3.10/site-packages/aiogram_dialog/window.py", line 74, in render
    current_data = await self.load_data(dialog, manager)
  File "/mnt/Trash/artem/PythonDev/aiogram_dialog_middlwares_issue/venv/lib64/python3.10/site-packages/aiogram_dialog/window.py", line 59, in load_data
    return await self.getter(**manager.data)
  File "/mnt/Trash/artem/PythonDev/aiogram_dialog_middlwares_issue/venv/lib64/python3.10/site-packages/aiogram_dialog/widgets/data/data_context.py", line 36, in __call__
    return await self.normal_getter(dialog_manager=dialog_manager, **kwargs)
  File "/mnt/Trash/artem/PythonDev/aiogram_dialog_middlwares_issue/dialog.py", line 18, in getter
    service: HelloWorldService = dialog_manager.data["hello_world_service"]  # NOT OK
KeyError: 'hello_world_service'
^CINFO:aiogram.dispatcher:Polling stopped
```

**Comment by t3m8ch** [comment]
Traceback when /dialog2:
```
<service.HelloWorldService object at 0x7f8a97bd9cf0>
INFO:aiogram.event:Update id=840847704 is handled. Duration 434 ms by bot id=1589390197
INFO:aiogram.event:Update id=840847705 is not handled. Duration 0 ms by bot id=1589390197
ERROR:aiogram.event:Cause exception while process update id=840847705 by bot id=1589390197
KeyError: 'hello_world_service'
Traceback (most recent call last):
  File "/mnt/Trash/artem/PythonDev/aiogram_dialog_middlwares_issue/venv/lib64/python3.10/site-packages/aiogram/dispatcher/dispatcher.py", line 238, in _process_update
    response = await self.feed_update(bot, update, **kwargs)
  File "/mnt/Trash/artem/PythonDev/aiogram_dialog_middlwares_issue/venv/lib64/python3.10/site-packages/aiogram/dispatcher/dispatcher.py", line 104, in feed_update
    response = await self.update.wrap_outer_middleware(self.update.trigger, update, kwargs)
  File "/mnt/Trash/artem/PythonDev/aiogram_dialog_middlwares_issue/venv/lib64/python3.10/site-packages/aiogram/dispatcher/middlewares/error.py", line 24, in __call__
    return await handler(event, data)
  File "/mnt/Trash/artem/PythonDev/aiogram_dialog_middlwares_issue/venv/lib64/python3.10/site-packages/aiogram/dispatcher/middlewares/user_context.py", line 23, in __call__
    return await handler(event, data)
  File "/mnt/Trash/artem/PythonDev/aiogram_dialog_middlwares_issue/venv/lib64/python3.10/site-packages/aiogram/dispatcher/fsm/middleware.py", line 39, in __call__
    return await handler(event, data)
  File "/mnt/Trash/artem/PythonDev/aiogram_dialog_middlwares_issue/venv/lib64/python3.10/site-packages/aiogram_dialog/manager/manager_middleware.py", line 35, in __call__
    return await handler(event, data)
  File "/mnt/Trash/artem/PythonDev/aiogram_dialog_middlwares_issue/venv/lib64/python3.10/site-packages/aiogram_dialog/context/intent_filter.py", line 67, in __call__
    return await handler(event, data)
  File "/mnt/Trash/artem/PythonDev/aiogram_dialog_middlwares_issue/venv/lib64/python3.10/site-packages/aiogram/dispatcher/event/telegram.py", line 239, in trigger
    return await wrapped_inner(event, kwargs)
  File "/mnt/Trash/artem/PythonDev/aiogram_dialog_middlwares_issue/venv/lib64/python3.10/site-packages/aiogram/dispatcher/event/handler.py", line 43, in call
    return await wrapped()
  File "/mnt/Trash/artem/PythonDev/aiogram_dialog_middlwares_issue/venv/lib64/python3.10/site-packages/aiogram/dispatcher/dispatcher.py", line 205, in _listen_update
    return await self.propagate_event(update_type=update_type, event=event, **kwargs)
  File "/mnt/Trash/artem/PythonDev/aiogram_dialog_middlwares_issue/venv/lib64/python3.10/site-packages/aiogram/dispatcher/router.py", line 128, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
  File "/mnt/Trash/artem/PythonDev/aiogram_dialog_middlwares_issue/venv/lib64/python3.10/site-packages/aiogram/dispatcher/router.py", line 124, in _wrapped
    return await self._propagate_event(
  File "/mnt/Trash/artem/PythonDev/aiogram_dialog_middlwares_issue/venv/lib64/python3.10/site-packages/aiogram/dispatcher/router.py", line 137, in _propagate_event
    response = await observer.trigger(event, **kwargs)
  File "/mnt/Trash/artem/PythonDev/aiogram_dialog_middlwares_issue/venv/lib64/python3.10/site-packages/aiogram/dispatcher/event/telegram.py", line 239, in trigger
    return await wrapped_inner(event, kwargs)
  File "/mnt/Trash/artem/PythonDev/aiogram_dialog_middlwares_issue/main.py", line 17, in di_middleware
    return await handler(event, data)
  File "/mnt/Trash/artem/PythonDev/aiogram_dialog_middlwares_issue/venv/lib64/python3.10/site-packages/aiogram/dispatcher/event/handler.py", line 43, in call
    return await wrapped()
  File "/mnt/Trash/artem/PythonDev/aiogram_dialog_middlwares_issue/venv/lib64/python3.10/site-packages/aiogram_dialog/dialog.py", line 155, in _callback_handler
    await window.process_callback(cleaned_callback, self, dialog_manager)
  File "/mnt/Trash/artem/PythonDev/aiogram_dialog_middlwares_issue/venv/lib64/python3.10/site-packages/aiogram_dialog/window.py", line 69, in process_callback
    await self.keyboard.process_callback(c, dialog, manager)
  File "/mnt/Trash/artem/PythonDev/aiogram_dialog_middlwares_issue/venv/lib64/python3.10/site-packages/aiogram_dialog/widgets/kbd/button.py", line 27, in process_callback
    await self.on_click.process_event(c, self, manager)
  File "/mnt/Trash/artem/PythonDev/aiogram_dialog_middlwares_issue/venv/lib64/python3.10/site-packages/aiogram_dialog/widgets/widget_event.py", line 20, in process_event
    await self.callback(event, source, manager, *args, **kwargs)
  File "/mnt/Trash/artem/PythonDev/aiogram_dialog_middlwares_issue/dialog.py", line 25, in on_click
    service: HelloWorldService = manager.data["hello_world_service"]  # NOT OK
KeyError: 'hello_world_service'
^CINFO:aiogram.dispatcher:Polling stopped
```

**Comment by Tishka17** [comment]
Fixed in 2.0.0b2
